l3build-check.lua

```diff
@@ -691,7 +711,7 @@ end
 
 -- Run one of the test files: doesn't check the result so suitable for
 -- both creating and verifying
-function runtest(name, engine, hide, ext, pdfmode, breakout)
+function runtest(name, engine, hide, ext, test_type, breakout)
```

Comment:
It would be a bit nicer to hide `ext` too in `test_type`, but it has to be separate to allow expectation files.